docs/UserGuide/Cluster/Cluster-Setup.md

```diff
@@ -19,268 +19,377 @@
+
+- `is_sync_enable` is always considered as `false`.
+
+  
+
 ## Start Service
-You can deploy a distributed cluster on multiple nodes or on a single machine, the main difference being that the latter needs to handle conflicts between ports and file directories.
-For detail descriptions, please refer to [Cluster Configuration Items](#Cluster Configuration Items).
+
+### Start Cluster
+
+You can deploy a distributed cluster on multiple nodes or on a single machine, the main difference being that the latter needs to handle  conflicts between ports and file directories. For detail descriptions, please refer to [Configurations](#Configurations).
 
 To start the service of one of the nodes, you need to execute the following commands:
 
-```bash
+```
 # Unix/OS X
 > nohup sbin/start-node.sh [printgc] [<conf_path>] >/dev/null 2>&1 &
 
 # Windows
 > sbin\start-node.bat [printgc] [<conf_path>]
 ```
-`printgc` means whether enable the gc and print gc logs when start the node,  
-`<conf_path>` use the configuration file in the `conf_path` folder to override the default configuration file.
 
-## OverWrite the configurations of Stand-alone node
+`printgc` means whether enable the gc and print gc logs when start the node,
+ `<conf_path>` use the configuration file in the `conf_path` folder to override the default configuration file.
 
-Some configurations in the iotdb-engines.properties will be ignored
+**If you start all the seed nodes, and all the seed nodes can contact each other without ip/port and file directory conflicts, the cluster has successfully started.**
 
-* `enable_auto_create_schema` is always considered as `false`. Use `enable_auto_create_schema` in 
- iotdb-cluster.properties to enable it, instead.
+### Cluster scalability
 
-* `is_sync_enable` is always considered as `false`.
+In the process of cluster running, users can add new nodes to the cluster or delete existing nodes. At present, it only supports node by node cluster scalability, and multi node cluster scalability can be transformed into a series of single node cluster scalability operations. The cluster will hanlde new cluster extension operations only after the last cluster scalability operation is completed.
 
-## Cluster scalability
-In the process of cluster running, users can add new nodes to the cluster or delete existing nodes. 
-At present, it only supports node by node cluster scalability, and multi node cluster scalability can be 
-transformed into a series of single node cluster scalability operations. The cluster will hanlde new 
-cluster extension operations only after the last cluster scalability operation is completed.
+**Start the following script on the new node to join the cluster to add a new node:**
 
-### add node
-Start the following script on the new node to join the cluster to add a new node:
-```bash
+```
 # Unix/OS X
 > nohup sbin/add-node.sh [printgc] [<conf_path>] >/dev/null 2>&1 &
-
+;
```

Comment:
now the start/add-node.sh do not support _[<conf_path>]_  parameter.

Reply:
The scripts can support [<conf_path>] parameter.